unstable/wlr-data-control-unstable-v1.xml

```diff
@@ -183,6 +183,74 @@
       <entry name="used_source" value="1"
         summary="source given to set_selection or set_primary_selection was already used before"/>
     </enum>
+
+    <!-- Version 3 additions -->
+
+    <event name="selection_serial" since="3">
+      <description summary="copy data to the selection">
+        This should be sent by compositors before a selection event with an
+        increasing serial number.
+
+        This can be used by the client in partnership to set_selection_response to indicate
+        pairing of a selection event to a reply.
+      </description>
+      <arg name="serial" type="uint">
```

Comment:
The arg tag is not closed.

Reply:
Yeah, that's probably what causes the CI failure.